app/controllers/admin/articles_controller.rb

```diff
@@ -7,8 +7,8 @@ def create
     @article = Article.new(article_params)
     
     if @article.save
-      redirect_to admin_articles_path
-      flash.notice = 'You have successfully created a new article!'
+      redirect_to admin_articles_path, alert: 'You have successfully created a new article!'
```

Comment:
You will only want to use alert for errors and notice for notifications.  Alerts should use https://getbootstrap.com/docs/4.1/components/alerts/ danger and notifications either info or success

If you see the API documentation I shared, it gives you an example of how to use notice with redirect_to